documentation/src/docs/asciidoc/release-notes/release-notes-5.8.0-M1.adoc

```diff
@@ -56,6 +56,8 @@ on GitHub.
 * Numeric literals used with `@CsvSource` or `CsvFileSource` can now be expressed using
   underscores as in some JVM languages, to improve readability of long numbers like
   `700_000_000`.
+* Top-level test classes can now be ordered by passing a fully-qualified class-name implementing
+  `ClassOrderer` as value of the new property `junit.jupiter.testclass.order.default`.
```

Comment:
```suggestion
* Top-level test classes can now be ordered by passing the fully-qualified name of a class
  implementing `ClassOrderer` as value of the new `junit.jupiter.testclass.order.default`
  configuration property.
```

Reply:
Do you want to apply this change and rebase or shall I take it from here?

Reply:
I just pushed the latest suggestion, rebased on main and squash various commits into one